install-java.sh

```diff
@@ -17,22 +17,27 @@
 # Installation script for setting up Java on Linux
 # ----------------------------------------------------------------------------
 
+java_bash_profile="/etc/profile.d/jdk.sh"
+
 java_dist=""
 java_dir=""
+say_yes_no="" # you can make 'no' a default to avoid prompt altogether
 
 function help {
     echo ""
     echo "Usage: "
-    echo "install-java.sh -f <java_dist> [-p] <java_dir>"
+    echo "install-java.sh [-y] -f <java_dist> [-p <java_dir>]"
```

Comment:
I'm okay with this change.

Reply:
However, this was already supported with `yes` command. https://github.com/chrishantha/install-java/blob/a6d6b17acd2a0ba6a550dcb21208b9e3bea48353/README.md#automate-java-installation